scripts/setup/rewrite_yaml_files.sh

```diff
@@ -0,0 +1,346 @@
+sed -e "s/_master_node_name_/${master_node_name}/g" | sed -e 's/"v1.9.0"/devel/g' | sed -e 's/"1.9.0"/devel/g' | sed -e '$d' > serving-core-firecracker-yq.yaml
+
+
+# serving-crds.yaml
+
+cat serving-crds.yaml | sed -e 's/"v1.9.0"/devel/g' | sed -e 's/"1.9.0"/devel/g' > serving-crds-yq.yaml
+
+# serving-default-domain.yaml
+
+cat serving-default-domain.yaml | 
+    yq ' 
+    (
+        select
+        (
+            .spec.template.metadata.labels.app == "default-domain"
+        ) | .spec.template.spec.containers[0].image
+    ) = "ghcr.io/vhive-serverless/default-domain-dd6f55160290a90018ba318b624be12e@sha256:0206e792ee18591903e8b8aaaba92d04548c9b63d9742a98195037697bdbc42e"' |
+sed -e 's/"v1.9.0"/devel/g' | sed -e 's/"1.9.0"/devel/g' | sed -e '$d' > serving-default-domain-yq.yaml
+
+# serving-hpa.yaml
+
+cat serving-hpa.yaml |  
+    yq '
+    (
+        select
+        (
+            .spec.template.metadata.labels.app == "autoscaler-hpa"
+        ) | .spec.template.spec.containers[0].image
+    ) = "ghcr.io/vhive-serverless/autoscaler-hpa-85c0b68178743d74ff7f663a72802ceb@sha256:1797f6c16f0c65b0255651f82e85399ba210cc353044177ad2af159fc20e2122"' |
+sed -e 's/"v1.9.0"/devel/g' | sed -e 's/"1.9.0"/devel/g' | sed -e '$d' > serving-hpa-yq.yaml
+
+# serving-post-install-jobs.yaml
+
+cat serving-post-install-jobs.yaml |
+    yq '
+    (
+        select
+        (
+            .spec.template.metadata.labels.app == "storage-version-migration-serving"
+        ) | .spec.template.spec.containers[0].image
+    ) = "ghcr.io/vhive-serverless/migrate-242d0a35bf580c5b411a545d79618fbf@sha256:2d2159119a4c3bd8470a14a970835aa1f56a64250a9ff8bc59ea534175f8548b"' |
+sed -e 's/"v1.9.0"/devel/g' | sed -e 's/"1.9.0"/devel/g' | sed -e '$d' > serving-post-install-jobs-yq.yaml
+
+# serving-storage-version-migration.yaml
+
+cat serving-storage-version-migration.yaml |
+    yq '
+    (
+        select
+        (
+            .spec.template.metadata.labels.app == "storage-version-migration-serving"
+        ) | .spec.template.spec.containers[0].image
+    ) = "ghcr.io/vhive-serverless/migrate-242d0a35bf580c5b411a545d79618fbf@sha256:2d2159119a4c3bd8470a14a970835aa1f56a64250a9ff8bc59ea534175f8548b"' |
+sed -e 's/"v1.9.0"/devel/g' | sed -e 's/"1.9.0"/devel/g' | sed -e '$d' > serving-storage-version-migration-yq.yaml
+
+# metallb-ipaddresspool.yaml
+
+cat << EOF > metallb-ipaddresspool.yaml
+apiVersion: metallb.io/v1beta1
+kind: IPAddressPool
+metadata:
+  name: first-pool
+  namespace: metallb-system
+spec:
+  addresses:
+    - 10.200.3.4-10.200.3.24
+EOF
+
+# kube.json
+
+cat << EOF > kube.json
+{
+	"K8sVersion": "1.25.9",
+	"AlternativeImageRepo": "",
+	"ApiserverAdvertiseAddress": "",
+	"PodNetworkCidr": "10.168.0.0/16",
+	"PodNetworkAddonConfigURL": "https://raw.githubusercontent.com/vhive-serverless/vHive/main/configs/calico/canal.yaml",
+	"ApiserverPort": "6443",
+	"ApiserverToken": "",
+	"ApiserverTokenHash": ""
+}
+EOF
```

Comment:
Better to store them as files and copy to vhive instead of creating them from the script.